packages/safe-core-sdk/src/configuration/contracts.ts

```diff
@@ -11,30 +15,44 @@ export interface ContractNetworksConfig {
 export const defaultContractNetworks: ContractNetworksConfig = {
   // mainnet
   1: {
-    multiSendAddress: '0x8D29bE29923b68abfDD21e541b9374737B49cdAD'
+    multiSendAddress: '0x8D29bE29923b68abfDD21e541b9374737B49cdAD',
+    safeMasterCopyAddress: '0x6851D6fDFAfD08c0295C392436245E5bc78B0185',
+    safeProxyFactoryAddress: '0x76E2cFc1F5Fa8F6a5b3fC4c8F4788F0116861F9B'
   },
   // rinkeby
   4: {
-    multiSendAddress: '0x8D29bE29923b68abfDD21e541b9374737B49cdAD'
+    multiSendAddress: '0x8D29bE29923b68abfDD21e541b9374737B49cdAD',
```

Comment:
can you get this info from safe-deployments package?

Reply:
Here :) https://github.com/gnosis/safe-core-sdk/issues/25